bokeh/models/markers.py

```diff
@@ -112,72 +121,72 @@
 #-----------------------------------------------------------------------------
 
 def Asterisk(*args, **kwargs):
-    ''' Render asterisk '*' markers. '''
+    ''' Render asterisk '*' markers. (deprecated) '''
```

Comment:
I have added "(deprecated)" to all the docstrings here. However, it would be nice to use some mechanism to automatically also add the information about the respective `Scatter` attribute that replaces each of the deprecated functions. Would it make sense to use the `deprecated()` function for this?

Reply:
This is an area for improvement. The current `deprecated` function is only for raising runtime warnings when the deprecated thing is *used*. It does not do anything to e.g. automatically add notes to docstrings. We used to have something like that but it got removed during some refactor. It's probably worth revisiting various kinds of docstring decorators like "deprecated in" and "added in" etc. But for now the docstrings will have have to be edited manually.

Reply:
I have opened issue #10889 for this.